### ISSUE TITLE
iOS screen size seems wrong on iPad

### ISSUE THREAD

**Issue by mrhelmut** [MonoGame/MonoGame#7768]
iPad backbuffer resolution seems initialized to a wrong value which doesn't seem to be the full screen. It results in iPadOS detecting a MonoGame app as being non-fullscreen and shows controls to set it to full screen.

It seems that the iOS implementation gets its screen size from two different code paths:

https://github.com/MonoGame/MonoGame/blob/80071c07b25aeda2d2a70a7ce61c571867371947/MonoGame.Framework/Graphics/PresentationParameters.cs#L186

https://github.com/MonoGame/MonoGame/blob/80071c07b25aeda2d2a70a7ce61c571867371947/MonoGame.Framework/Platform/iOS/iOSGameView.cs#L216

```UIScreen.MainScreen.Bounds``` and ```UIView.Bounds``` might not return the same result.

#### What version of MonoGame does the bug occur on:
- MonoGame 3.8

#### What operating system are you using:
- Windows

#### What MonoGame platform are you using:

- iOS

**Comment by mrhelmut** [comment]
Actually the size handling difference I pointed seems normal. It's a bit misleading, but on iOS not everything happens on the GraphicsDevice, some of the stuff like the framebuffer creation is cascaded to the UIView after size event changes.

So the issue is likely higher up.

**Comment by Mindfulplays** [comment]
I found out that if you deploy an iPhone app as an iPad app then the controls show up "2x". Mostly for very old SDK level.
Which iOS version / iPad version is this? And what do those controls look like (just to make sure)?
FWIW, I haven't noticed the screen size being wrong - I verified by dumping the screen size and confirming against official screen size list.

**Comment by mrhelmut** [comment]
This is a mockup of what I'm getting. The blue is MonoGame's ```Viewport````. I would expect the blue to fill the entire screen and I wouldn't expect the battery indicator nor the arrows to go full screen.
I don't have this problem on iPhone (it's an universal iOS app). I have ```UIRequiresFullScreen``` set to ```true```. I'm targeting the latest iOS with a backcompat of iOS 11.

![image](https://user-images.githubusercontent.com/3427451/166001187-b5fe0cbc-7c6b-4d81-9d4a-db6d81e5423f.png)

My guess is that the ```UIView``` isn't filling the screen and makes iPadOS to show the other controls.

Also, I'm changing ```RenderTarget``` a few times with different sizes during the rendering, so maybe it has something to do with the framebuffer not restoring properly its intended. Or maybe changing RenderTarget affects the UIView while it shouldn't.
(I believe it has something to do with RenderTargets more than the UIView initialization, because the size of the blue rectangle is perfectly 16:9, which is what is the size of the RenderTargets I'm using).

**Comment by Mindfulplays** [comment]
Thanks, I don't think it's the `UIView` but likely the app `Info.plist` itself.

[Link](https://support.apple.com/guide/ipad/zoom-an-app-to-fill-the-screen-ipade8a4de65/ipados) describes the condition in which this control shows (as I mentioned it's likely a version / iPhone->iPad app compatibility problem).

You could actually try removing all of MG stuff and still be able to repro it.

If you do have `Info.plist` snippet, it might help debug this issue.

**Comment by mrhelmut** [comment]
It might be it! Harry pointed me to this that I'm missing:

```
    <key>UIDeviceFamily</key>
    <array>
        <integer>1</integer>
        <integer>2</integer>
    </array>
```

Which makes sense... iPadOS is likely trying to run the game as an iPhone app...

I'll try that on Monday, and will likely update the iOS template to avoid future confusion.

**Comment by mrhelmut** [comment]
Should be fixed by #7771 

I'll verify that on Monday.

**Comment by mrhelmut** [comment]
Confirmed fixed. Thanks for the insights.
